src/natcap/invest/wave_energy.py

```diff
@@ -139,14 +139,14 @@
         },
         "analysis_area_path": {
             "type": "option_string",
-            "options": [
-                "West Coast of North America and Hawaii",
-                "East Coast of North America and Puerto Rico",
-                "North Sea 4 meter resolution",
-                "North Sea 10 meter resolution",
-                "Australia",
-                "Global"
-            ],
+            "options":
+                "West Coast of North America and Hawaii": "",
+                "East Coast of North America and Puerto Rico": "",
+                "North Sea 4 meter resolution": "",
+                "North Sea 10 meter resolution": "",
+                "Australia": "",
+                "Global": ""
+            },
```

Comment:
I just noticed this looks like a syntax error with no opening `{`?

Reply:
Oh, that must be a link to an older commit. In the latest version it's correct: https://github.com/emlys/invest/blob/f3e9f25de146af3aca1532c9c7fef589a738afbb/src/natcap/invest/wave_energy.py#L150